### PR TITLE
Rewrite of the upstream cache control headers docs

### PR DESCRIPTION
**Pull request by titpetric**
## Related Issue (if applicable)

https://tyktech.atlassian.net/browse/TT-9022

## Description

Better documents upstream cache behaviour. Editing for clarity, info...

## Motivation and Context

Adoption problems due to bad docs.

## Preview Link

https://deploy-preview-2596--tyk-docs.netlify.app/nightly/docs/basic-config-and-security/reduce-latency/caching/#upstream-control

## Types of changes

- [ ] Fixing typo (please merge to production)
- [ ] Documenting a new feature (please merge to production)
- [ ] Documentation for future release (please do not merge to production)
- [ ] Something else (please add if needs merging to production or not)

## Checklist

- [ ] I have [reviewed the guidelines](../CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](../CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I have added a preview link.

### CODE REVIEW

File: tyk-docs/content/basic-config-and-security/reduce-latency/caching.md [hunk 1]
```diff
@@ -138,30 +138,70 @@ Go to the Endpoint Designer tab. From the path you want to cache, select the **C
...
+
+Explanation:
+
+**x-tyk-cache-action-set**: If the header is set to `1`, Tyk caches the
+request, even for non-safe requests. If the header is empty or absent,
+Tyk follows its default behavior, which typically involves not caching
+the request, or caching only valid response codes
+(`cache_response_codes`).
+
+**x-tyk-cache-action-set-ttl**: If the header is present, Tyk uses the
```
Comment (letzya) [inline tyk-docs/content/basic-config-and-security/reduce-latency/caching.md:185]: if I do ctrl+f for "x-tyk-cache-action-set-ttl" - i see this info for the two settings twice. Please check

**Review by letzya — CHANGES_REQUESTED** [review]
Per comment. needs a bit of tidying up.
I also removed the DX writer and added a peer as a reviewer. Add Simon after it's approved.

File: tyk-docs/content/basic-config-and-security/reduce-latency/caching.md (hunk 1 above)
Comment (titpetric) [reply tyk-docs/content/basic-config-and-security/reduce-latency/caching.md:185]: it's referenced twice, one is the setting, one is the behaviour explanation

**Comment by titpetric** [comment]
@andyo-tyk will open a new proposal, closing this PR.

**Pull request closed without merging**